src/react-slidy-slider.jsx

```diff
@@ -67,10 +72,6 @@ export default class ReactSlidySlider extends Component {
     this.slidyInstance && this.slidyInstance.destroy()
   }
 
-  shouldComponentUpdate (nextProps) {
```

Comment:
That's being done because render optimizations. React Slidy is not a normal React component because as you can see the logic is in a pure javascript file instead inside the components. The React component is only wrapping that and abstracting from the developer in order to easily use the slider on React projects. Said that, the `shouldComponentUpdate` is always `false` in order to avoid React messing with the performance of the slider.